code/game/objects/items/ropechainbola.dm

```diff
@@ -8,6 +8,7 @@
 	slot_flags = ITEM_SLOT_HIP|ITEM_SLOT_WRISTS|ITEM_SLOT_NECK|ITEM_SLOT_BELT
 	throwforce = 3
 	w_class = WEIGHT_CLASS_SMALL
+	grid_width = 32 //1x2 item, chains too
```

Comment:
This should be an atomised change

Reply:
I'm still learning how to work github, and reverting a single file seems very difficult for some reason. Turns out you're supposed to make a bunch of commits so this doesn't happen. I can just delete the lines I added in a new commit, but I'm not sure if that's what you want.

Reply:
well commits make reverting things easy but in this case you can just delete the line and make another pr for it with a new branch